CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ SET(${PROJECT_NAME}_SRCS
     ndicapi_serial.h
     ndicapi.h
     ndicapi_socket.h
+    ${CMAKE_CURRENT_BINARY_DIR}/ndicapiExport.h
```

Comment:
Does `add_library` complain that the file doesn't exist until after configuration?

Reply:
I understand that, what I meant was: this file does not exist until `configure_file` (line 141) is run later in this file, but `add_library` (line 91) will look for it before it's created. Does it complain about a missing file?

Reply:
Sorry for the misunderstanding. No failure on my side but I think it will be safer to add an existing file. I'll update the PR ASAP.